src/open_inwoner/pdc/tests/test_views.py

```diff
@@ -242,11 +242,11 @@ def test_category_detail_description_rendered(self):
         response = self.client.get(url)
 
         self.assertIn(
-            '<p class="p">A <em>descriptive</em> description</p>',
+            "A <em>descriptive</em> description",
```

Comment:
@Bartvaderkin This test `test_category_detail_description_rendered`  we are not sure about. I can't replace the `p` class with the NLDesignSystem class here (since then this test doesn't not pass). But if I remove the class entirely it does pass. @pi-sigma wasn't sure about this assertion.

Reply:
@jiromaykin I'm not quite sure. I think I would just remove the opening `<p .. >` (and leave the closing one).

````py
self.assertIn("A <em>descriptive</em> description</p>", response.rendered_content)
````

And the second assertion (the `self.assertNotIn(` seems a bit odd and probably could be removed (not quite sure why it is there, maybe @pi-sigma remembers).

Reply:
I've adjusted/removed them, so this is ready for review.